.github/workflows/validate-issues.yml

```diff
@@ -6,13 +6,14 @@ on:
 
 jobs:
   check-labels-on-issues:
-    uses: Automattic/dangermattic/.github/workflows/reusable-check-labels-on-issues.yml@v1.0.0
+    uses: Automattic/dangermattic/.github/workflows/reusable-check-labels-on-issues.yml@v1.1.2.1
```

Comment:
I don't see this tag in the repo https://github.com/Automattic/dangermattic/tags but I assume it's here because we don't have a way to test the behavior until it lands on the main branch and it would be too cumbersome to point to the WIP branch from https://github.com/Automattic/dangermattic/pull/82, verify, ship the tag, then open a new PR pointing to the tag.

Reply:
Exactly.
As this update will contain only the workflow change, there's also no need to ship a full fledged version I think, so I'll just tag it once https://github.com/Automattic/dangermattic/pull/82 lands the so that GitHub Actions can find the new version.